src/results/result-form-field.component.tsx

```diff
@@ -24,9 +25,15 @@ const ResultFormField: React.FC<ResultFormFieldProps> = ({
 
   return (
     <>
+      {Object.keys(errors).length > 0 && (
+        <div className={styles.errorDiv}>All fields are required</div>
```

Comment:
Can this be driven by some sort of configuration? This seems like something that might not be the case for every scenario

Reply:
kindly give an example? what kind of configuration?